working-groups/TSC working groups.md

```diff
@@ -0,0 +1,3 @@
+## TSC working groups 
+
+Have a look at the ['TSC working groups' document.[(https://docs.google.com/document/d/1_7LffEtgvZKK5rj1ucfxzU1sX2ma8qVlW9AH_I3X1io/edit)]
```

Comment:
Bad rendering. Also avoid using spaces in filenames. 

(My best suggestion would be to not refer this file as of now)

Reply:
https://github.com/moja-global/About_moja_global/pull/155#issuecomment-1030716694